Makefile

```diff
@@ -35,9 +35,9 @@ run:
 	cargo run --features "${ENABLE_FEATURES}"
 
 release:
-	cargo build --release --features "${ENABLE_FEATURES}"
+	cargo build --release --features "${ENABLE_FEATURES} no-fail"
```

Comment:
I prefer using a var to control the fail build like RocksDB does.

E.g., we can support a `fail_release` to build.